service/src/main/resources/application.properties

```diff
@@ -2,7 +2,7 @@ spring.datasource.url=
 spring.datasource.username=
 spring.datasource.password=
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
-spring.mvc.throw-exception-if-no-handler-found=true
+spring.mvc.throw-exception-if-no-handler-found=false
```

Comment:
I guess i assumed that the "throw exception" meant 404 so wasn't sure what would happen if that was false.  But 404 makes perfect sense.